pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -4685,6 +4685,29 @@ def test_all_offset_classes(self, tup):
         assert first == second
 
 
+# ---------------------------------------------------------------------
+
```

Comment:
prob move to liboffsets tests?

also ``_rd_kwds`` event internally is pretty hard to grok. you can de-privatize and name it more descriptively.